modules/fluid_properties/doc/content/modules/fluid_properties/sqa/fluid_properties_sdd.md

```diff
@@ -1 +1,66 @@
 !template load file=sqa/module_sdd.md.template category=fluid_properties module=Fluid Properties
+
+!template! item key=introduction
+The [!ac](MOOSE) {{module}} module is based on the MOOSE framework and thus inherits
+the unique features and base characteristics of the framework, as outlined in the [framework_sdd.md].
+Specific details unique to the module are outlined in this document.
```

Comment:
Would welcome your feedback on whether this intro or something like it should just go into the higher-level template. Left it open in case leads wanted to place something custom here.

Reply:
Something like this happens several times in the EM example - could create more fields (`keys`) in a follow-up PR to reduce truly boiler-plate info.